render_engine/page.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, **kwargs) -> None:
         self.slug = slugify(self.slug)
 
     @property
-    def url(self) -> str:
+    def url(self) -> Path:
```

Comment:
@kjaymiller Is this correct? I updated this based on the return type, but I wasn't sure if the changed return type was actually a bug in the function implementation and intended to be `str`.

Reply:
I haven't been testing types and honestly I think I wrap it in a Path so either will work. technically it would be `typing.Any[str, Path]`